config/locales/ar.yml

```diff
@@ -78,6 +78,502 @@ ar:
     bank_holiday_benefits_html:
```

Comment:
I think we might have copied too many keys in this commit? They don't look strictly limited to html_publications keys.

Reply:
Which keys are you referring to? I don't think line 78 was added by this PR?